include/zephyr/arch/x86/ia32/scripts/shared_kernel_pages.ld

```diff
@@ -48,6 +48,7 @@
 #endif /* CONFIG_X86_USERSPACE */
 	. += (GDT_NUM_ENTRIES + GDT_NUM_TLS_ENTRIES) * 8;
 #endif /* LINKER_ZEPHYR_FINAL */
+	. += CONFIG_GDT_RESERVED_NUM_ENTRIES * 8;
```

Comment:
Should this be inside the `#ifdef LINKER_ZEPHYR_FINAL..#endif` block like the other gdt entries?

Reply:
Done. Put is into LINER_ZEPHYR_FINAL block same as other gdt entries, please review.